recipes/pint-xarray/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pint-xarray" %}
+{% set version = "0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a48be363f2cd870c0c73e11b9e2b04160c1cfb089fee98277189a3afc26ecc45
+
+build:
+  number: 0
+  noarch: python
+  script: SETUPTOOLS_SCM_PRETEND_VERSION={{ version }} {{ PYTHON }} -m pip install . -vv
```

Comment:
Do we really need the `SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}`?

Reply:
last time I checked, `setuptools_scm` would fail to detect the version and fall back to `0.0.0` (see conda-forge/pint-feedstock#34).

Reply:
`setuptools_scm` should write the version number into the `sdist` metadata. There should be no need for overwriting it or using `write_to`.